### PR TITLE
changed the template to use managed disks

### PR DESCRIPTION
**Pull request by ramankumarlive**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

@singhkays

### CODE REVIEW

File: 101-vm-user-image-data-disks/azuredeploy.json [hunk 1]
```diff
@@ -41,107 +61,125 @@
...
+    "apiVersion": "2015-06-15",
+    "templatelink": "[concat('https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/101-vm-user-image-data-disks/', parameters('newOrExistingVnet'),'vnet.json')]"
   },
   "resources": [
     {
-      "type": "Microsoft.Storage/storageAccounts",
-      "name": "[variables('diagStorageAccountName')]",
-      "apiVersion": "2016-01-01",
+      "type": "Microsoft.Compute/images",
+      "apiVersion": "2016-04-30-preview",
```
Comment (singhkays) [inline 101-vm-user-image-data-disks/azuredeploy.json]: Can you use the latest api version 2017-03-30 ?

File: 101-vm-user-image-data-disks/azuredeploy.json [hunk 2]
```diff
@@ -153,20 +191,21 @@
...
+                "id": "[reference('vnet-template').outputs.subnet1Ref.value]"
               }
             }
           }
         ]
       }
     },
     {
-      "apiVersion": "[variables('apiVersion')]",
+      "apiVersion": "2016-04-30-preview",
```
Comment (singhkays) [inline 101-vm-user-image-data-disks/azuredeploy.json]: 2017-03-30

File: 101-vm-user-image-data-disks/metadata.json [hunk 3]
```diff
@@ -2,6 +2,6 @@
   "itemDisplayName": "Create a VM from User Image",
   "description": "This template allows you to create a Virtual Machines from a User image. This template also deploys a Virtual Network, Public IP addresses and a Network Interface.",
   "summary": "Create a Virtual Machine from a User Image",
-  "githubUsername": "iandow",
-  "dateUpdated": "2015-12-22"
+  "githubUsername": "slapointe",
```
Comment (singhkays) [inline 101-vm-user-image-data-disks/metadata.json]: This should be your username?

File: 101-vm-user-image-data-disks/azuredeploy.json (hunk 2 above)
Comment (ramankumarlive) [reply 101-vm-user-image-data-disks/azuredeploy.json]: done

File: 101-vm-user-image-data-disks/metadata.json (hunk 3 above)
Comment (ramankumarlive) [reply 101-vm-user-image-data-disks/metadata.json]: done

File: 101-vm-user-image-data-disks/azuredeploy.json (hunk 1 above)
Comment (ramankumarlive) [reply 101-vm-user-image-data-disks/azuredeploy.json]: done

**Comment by singhkays** [comment]
lgtm, merged

**Pull request merged**